src/widgets/unfold_widgets/preprocessing_widget.py

```diff
@@ -237,5 +234,27 @@ def render_reduction_group(self, max_dimensions):
         self.auto_reduction.setDisabled(max_dimensions < 3)
 
     def show_reduction_results(self):
-        self.preview_screen = DataPreviewScreen(self)
+        self.preview_screen = DataPreviewScreen(self, title="Reduction results", reason=PreviewReason.REDUCTION)
+        self.preview_screen.show()
+
+    def render_estimation_group(self):
+        self.estimate_group.setTitle("Estimate missing values")
```

Comment:
Since only one of the option is using estimates, maybe it would be better to name the group `Fill missing values` or `Handle missing data` and give a third option to drop rows with missing values